chunk/tag.go

```diff
@@ -216,3 +218,57 @@ func decodeInt64Splice(buffer *[]byte) int64 {
 	*buffer = (*buffer)[n:]
 	return val
 }
+
+// marshall friendly tag structure
+type MarshallTag struct {
```

Comment:
I think that this type can be unexported.

Reply:
Done

Reply:
Thanks.